src/pages/blog/community-day.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: '../../layouts/Post.astro'
+title: Community Day
+description: 'After a long week of new features and releases, we wanted to  turn our attention back to the most important aspect of this project: the Astro community.'
```

Comment:
@FredKSchott I pulled this from the first paragraph but you may want to update it

Reply:
Oh good catch, I'd use the Twitter catchy first tweet instead:

description: 'Happy Astro Contributor Day! Who are some of our awesome contributors? Where does Astro's sponsorship 💰 go? How does Astro give back to OSS? Lets get into it!'

also: title should be "Astro Contributor Day" or "Happy Astro Contributor Day!", whichever you think is best.